controller/service.rst

```diff
@@ -66,36 +66,39 @@ Then you can define it as a service as follows:
 
         # app/config/services.yml
         services:
-            app.hello_controller:
+            AppBundle\Controller\HelloController:
                 class: AppBundle\Controller\HelloController
 
     .. code-block:: xml
 
         <!-- app/config/services.xml -->
         <services>
-            <service id="app.hello_controller" class="AppBundle\Controller\HelloController" />
+            <service id="AppBundle\Controller\HelloController" class="AppBundle\Controller\HelloController" />
         </services>
 
     .. code-block:: php
 
         // app/config/services.php
         use AppBundle\Controller\HelloController;
 
-        $container->register('app.hello_controller', HelloController::class);
+        $container->register(HelloController::class, HelloController::class);
 
 Referring to the Service
 ------------------------
 
-To refer to a controller that's defined as a service, use the single colon (:)
-notation. For example, to forward to the ``indexAction()`` method of the service
-defined above with the id ``app.hello_controller``::
+If the fully-qualified class name (FQCN) of your controller is also the id of
+your service then you can refer to your controller using the usual notations.
+For example, to forward to the ``indexAction()`` method of the service
+defined above with the id ``AppBundle\Controller\HelloController``::
 
-    $this->forward('app.hello_controller:indexAction', array('name' => $name));
+    $this->forward('AppBundle:Hello:index', array('name' => $name));
```

Comment:
Because this doesn't work with other service ids (this works thanks to https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php#L70).